recipe/meta.yaml

```diff
@@ -23,6 +23,20 @@ requirements:
   build:
     - python
     - pip
+    - futures               # [py2k]
+    - decorator
+    - tornado >=4
+    - python-dateutil >=2.1
+    - pyzmq >=13
+    - ipython_genutils
+    - ipython >=4
+    - jupyter_client
+    - ipykernel
```

Comment:
Why are all of these needed?

Reply:
runtime dependencies are needed for the  `nbextension install` command.